pkg/controller/workflow/reason.go

```diff
@@ -12,9 +12,11 @@ const (
 
 // Atlas Project reasons
 const (
-	ProjectNotCreatedInAtlas   ConditionReason = "ProjectNotCreatedInAtlas"
-	ProjectIPAccessInvalid     ConditionReason = "ProjectIPAccessListInvalid"
-	ProjectIPNotCreatedInAtlas ConditionReason = "ProjectIPAccessListNotCreatedInAtlas"
+	ProjectNotCreatedInAtlas                ConditionReason = "ProjectNotCreatedInAtlas"
+	ProjectIPAccessInvalid                  ConditionReason = "ProjectIPAccessListInvalid"
+	ProjectIPNotCreatedInAtlas              ConditionReason = "ProjectIPAccessListNotCreatedInAtlas"
+	ProjectPEServiceIsNotReadyInAtlas       ConditionReason = "ProjectPrivateEndpointServiceIsNotReadyInAtlas"
+	ProjectPrivateEndpointNotCreatedInAtlas ConditionReason = "ProjectPrivateEndpointNotCreatedInAtlas"
```

Comment:
Is this "Not Created" or "We tried to create it but got an error"? - the name of the condition is not so important but want to make sure we understand the workflow and what these will really mean if customers see them.

Reply:
Good point. I will think about these names. Keep in mind that there is also a message for each such error which is more descriptive.